torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -589,6 +589,14 @@ class CompiledBackward(GenericCompiledBackward[CompiledFxGraph], FxGraphCacheLoa
     def _is_backward(self) -> bool:
         return True
 
+    def post_compile(
```

Comment:
I wanted to put this in GenericCompiledBackward, but the post_compile its referencing in super() needs to refer to FxGraphCacheLoadable's post_compile, so I need to copy it twice for Bundled vs. non Bundled. Multiple inheritance is confusing 😅